fhirAssembly/build.gradle

```diff
@@ -0,0 +1,213 @@
+plugins {
+    id 'java'
+    id("io.freefair.lombok") // version "6.3.0" // version "8.7.1"
+}
+
+group 'io.vantiq'
+version 'unspecified'
+
+java {
+    sourceCompatibility = 1.11
+}
+
+ext {
+    fhirServerUrl = 'http://localhost:8090/fhir'
+    fhirServerStartLimitSeconds = 180
+}
+
+repositories {
+    mavenCentral()
+
+    maven {
+        url "https://vantiqmaven.s3.amazonaws.com/"
+        metadataSources {
+            // The artifact() metadataSource allows the dependency resolver to use the existence of the file
+            // we want to be sufficient.  Then we don't have to publish fake POM files for our coco models
+            // or image/video files.  Since this repo is used primarily to store big test objects that
+            // we don't want in our GIT repo, this allows our strategy to work.  This change came during
+            // upgrade to Gradle 6.x from 5.x.
+            mavenPom()
+            artifact()
+        }
+    }
+}
+
+test {
+    // set heap size for the test JVM(s)
+    minHeapSize = "2048m"
+    maxHeapSize = "2048m"
+}
+
+tasks.withType(Test).configureEach {
+    // Use the build dir as a base to get our various test artifacts.
+    systemProperty 'buildDir', layout.buildDirectory.get().asFile.absolutePath
+    if (rootProject.hasProperty("TestAuthToken")) {
+        systemProperty "TestAuthToken", rootProject.findProperty("TestAuthToken") ?: "empty"
+    }
+    if (rootProject.hasProperty("TestVantiqServer")) {
+        systemProperty "TestVantiqServer", rootProject.findProperty("TestVantiqServer") ?: "empty"
+    }
+}
+
+configurations {
+    testResources.extendsFrom(testRuntimeOnly) // added so configurations search is allowed
+}
+
+dependencies {
+    testImplementation "org.slf4j:slf4j-api:1.7.25"
+    testImplementation "org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}"
+    testImplementation "org.projectlombok:lombok:${lombokVersion}"
+    testImplementation 'com.fasterxml.jackson.core:jackson-databind:2.17.2'
+
+    // Used for uploading documents (images) to VANTIQ with VANTIQ SDK
+    testImplementation "io.vantiq:vantiq-sdk:${vantiqSDKVersion}"
+
+    // Use JUnit test framework
+    testImplementation 'junit:junit:4.13.1'
+    // Data to preload into our FHIR server
+    testResources "vantiq.testArtifacts:fhirR4Sample:1.0@zip"
+}
+
```

Comment:
We've found some sample data in FHIR R4 form.  We've taken a small subset of this & include this as the dependency above.  The `unzip...` and `upload...` tasks load this data into our server.  Since the server is configured as in-memory only, we do this at startup.